config/prod.exs

```diff
@@ -27,3 +27,5 @@ config :realtime_signs,
   external_config_getter: ExternalConfig.S3,
   sign_updater_mod: MessageQueue,
   restart_fn: &System.restart/0
+
+config :realtime_signs, RealtimeSignsWeb.Endpoint, secret_key_base: System.fetch_env!("SECRET_KEY_BASE")
```

Comment:
This file is evaluated at *build* time, so this won't work. The `SECRET_KEY_BASE` env var is (I'm assuming) in the WinSW config, which is only available at run time.

`config/runtime.exs` is the special file specifically for configuring things at runtime, often with environment variables. So this line should go there, wrapped in an `if config_env() == :prod`.

Reply:
Ah gotcha. Just moved it over to `runtime.exs`